### PR TITLE
chore: add my notes on chapter 2 of working the command line

### LINKED ISSUE #24: chore: add my notes on chapter 2 of working the command line
I have some notes gathered throughout the reading that I'd like to contribute with.

### PR DESCRIPTION
**Pull request by bpires**
I have some complementary notes gathered throughout the reading that I'd like to add:
- Add Yarn as another package manager for Node
- Register the author's book suggestion on git

Fix #24

### CODE REVIEW

File: working-the-command-line/chapter2/index.md [hunk 1]
```diff
@@ -22,7 +22,10 @@ On macOS the most popular package manager is [`brew`](https://brew.sh/).
 
 - [Ruby gems](https://rubygems.org/)
 - [pip for Python](https://codeberryschool.com/blog/en/install-python-pip/)
-- [npm for Nodejs](https://npmjs.org/)
-
+- [npm for Nodejs](https://npmjs.org/) | [Yarn package manager](https://yarnpkg.com/)
```
Comment (schalkneethling) [inline working-the-command-line/chapter2/index.md:25]: I reckon it would be worth adding `pnpm` as well https://pnpm.io/

**Review by schalkneethling — COMMENTED** [review]
Thank you, @bpires. One suggestion.

File: working-the-command-line/chapter2/index.md (hunk 1 above)
Comment (bpires) [reply working-the-command-line/chapter2/index.md:25]: Great addition.

**Review by schalkneethling — APPROVED** [review]
Thanks, @bpires 🌮

**Pull request merged**